lib/spack/spack/schema/modules.py

```diff
@@ -99,6 +99,7 @@
                 'type': 'boolean',
                 'default': False
             },
+            'defaults': array_of_strings,
```

Comment:
I had to add "defaults" to the `spec_regex` above to avoid:
```
$ spack config get modules
==> Error: /home/culpo/.spack/modules.yaml:5: [] is not of type 'object'
```